src/routes/index.js

```diff
@@ -14,15 +14,20 @@ router.post('/users', catchErrors(userController.createUser));
 // Remove User using their ID
 router.delete('/user/:id', catchErrors(userController.removeUser));
 // Update User Info
-router.post('/user/:id', catchErrors(userController.updateUser));
+router.put('/user/:id', catchErrors(userController.updateUser));
 //Get Specific User
 router.get('/user/:id', catchErrors(userController.getUser));
 
 // More Routes
 //Get First Name
 router.get('/users/firstname/:id', catchErrors(userController.getUserFirstName));
 //Set First Name
-router.post('/users/firstname/:id', catchErrors(userController.setUserFirstname));
+router.put('/users/firstname/:id', catchErrors(userController.setUserFirstname));
+
+//Get Last Name
+router.get('/users/lastname/:id', catchErrors(userController.getUserLastName));
```

Comment:
Please correct these routes to REST standard, it should be  <resource>/<id>/<property>:

``router.get('/users/:id/lastname', ...);``

Reply:
Hey samuel,
Good catch.. Give me a sec to update this and push

Reply:
And I fixed that...